### DISCUSSION TITLE
Export only relevant articles

### DISCUSSION THREAD

**Discussion by AKoert** [asreview/asreview#1322] [Q&A]
Hi ASReview, 

I am very happy to use your software, as it makes the review process faster, thanks! Currently, we use ASReview as an initial screening method, in which we would like to switch to another software program to do an extensive selection with more authors. It would therefore be extremely helpful to be able to just export the articles I've marked as relevant. At the moment I am only able to export the entire dataset. 

Can someone help me with this?

Thanks in advance!

**Comment by J535D165** [comment]
Hi @AKoert, 

Currently, this feature is not available. You can export the results and make the subset in a reference manager. If the export format is RIS: you can search the records with `ASReview_relevant` in the notes field. For tabular datasets (like CSV, TAB, XLSX), you can do this in Excel or a similar program. 

Nevertheless, an excellent idea to add an option to export the subset only. I will create a feature request out of it, maybe somebody is interested in working on it.

**Reply by AKoert** [reply]
Hi! Thanks for the quick reply, would be amazing if this is a feature in the future. Hope this helps others as well!

**Reply by rohitgarud** [reply]
@AKoert hi.. In which file format do you want to export your records? If the .csv file format is okay for you, I can quickly add this feature to  the [ASReview notes export extension](https://github.com/rohitgarud/asreview-notes-export) so that you also get your notes with the exported relevant records such as reasons for inclusion

**Reply by AKoert** [reply]
Thanks a lot for this suggestion! I solved it by selecting the articles manually (I uploaded the CSV file in Rayyan software and all my "relevant" articles were on top, so it was an easy selection. After this I could export/copy the selected articles to start a dataset with just the relevant articles). Have a nice day!

**Reply by Rensvandeschoot** [reply]
great suggestions! @rohitgarud If you see a way to add it to the export functionality in ASReview? Same for the notes option, maybe there is a way to do this in ASReview LAB directly...

**Reply by rohitgarud** [reply]
Thank you @Rensvandeschoot. I will work on the backend part of this functionality, hoping to collaborate with someone working on the front end.

**Reply by Rensvandeschoot** [reply]
ping @terrymyc

**Reply by rohitgarud** [reply]
Hi @terrymyc.. can we add a checkbox in the export window (say `Export only relevant records`) when the dataset option is selected so that people can check it if they want to only export the relevant records. And how this boolean value can be accessed in the backend?
